formatters/stylish/stylish.js

```diff
@@ -0,0 +1,107 @@
+/*
+ * This file was forked from https://github.com/eslint/eslint/blob/v7.8.1/lib/cli-engine/formatters/stylish.js
+ * With a single modifcation on line 105 to output the number of files checked when all pass.
```

Comment:
Is this the reason we had fluid-grunt-eslint in the first place?

Regardless, it would be nice to figure out a way to keep this better up to date.  This is already two releases behind ESlint.  Maybe some kind of patch/build strategy with tests for the affected behaviour?

Reply:
Yes, this is basically the reason we had fluid-grunt-eslint; however, since then they've exposed the tooling to implement custom formatters. In this case we are only forking the formatter and using our own version based off of their default, stylish, formatter; we aren't forking eslint as a whole. Additionally the file this is forked from hasn't been updated in 13 months, so there is no need to update it at this point. 

I do take your point about not wanting to have a fork, for example incase some of the contract for using the formatter changes in the future. I have filed a ticket, https://github.com/eslint/eslint/issues/13725, with eslint to hopefully have the changes added there directly instead.

Reply:
I'm happy with that, we can review periodically as we update other proper dependencies from time to time.